static/src/javascripts/projects/common/modules/audio/ProgressBar.js

```diff
@@ -106,11 +106,17 @@ export default class ProgressBar extends Component<Props, State> {
     };
 
     update = (e: MouseEvent) => {
-        const position = e.clientX - this.state.left;
+        const rawPosition = e.clientX - this.state.left;
```

Comment:
You should probably use `e.offsetX` instead, as `clientX` is relative to the viewport.

Reply:
@regiskuckaertz do you mean `pageY`?  https://stackoverflow.com/questions/6073505/what-is-the-difference-between-screenx-y-clientx-y-and-pagex-y